app/models/cart.rb

```diff
@@ -0,0 +1,5 @@
+class Cart < ApplicationRecord
+    belongs_to :user
+    has_many :cart_placements
```

Comment:
best to add `dependent: :delete_all' here.  If you delete a cart, you want to delete the cart_placement records that go with it, or that table will grow without limit

Reply:
I put , dependent: :destroy on line three with the has_many :cart_placements.